climada/conf/climada.conf

```diff
@@ -79,5 +79,10 @@
         }
     },
     "log_level": "WARNING",
-    "max_matrix_size": 1000000000
+    "max_matrix_size": 1000000000,
+    "data_api": {
+        "host": "https://climada.ethz.ch",
+        "chunk_size": 8192,
```

Comment:
does this mean that the chunk_size is fixed, would it make sense to adapt it to the size of the files?

Reply:
No, but it could make sense to adapt it to the download capacity of the internet connection. However I really think it hardly matters. I haven't tested it but if I had to guess I'd say anything between a 1K and 1M would do and the performance loss/gain would be in the low percentages.